docusaurus/docs/reactnative/ui-components/reaction_list.mdx

```diff
@@ -91,6 +91,14 @@ Background color for reaction list container when [`alignment`](#alignment) is `
 
 <!-- ### strokeSize -->
 
+### reactionCounterColor
+
+Color for reaction's counter labels, default is 'white'.
+
+| Type   |
+| ------ |
+| string |
```

Comment:
Can we move this to theme instead please

theme > messageSimple > reactionList

Reply:
I don't know where exactly to move this as I'm not familiar with this structure of documentation. I can make the changes you requested if you could be more specific.